benchmark/buffers/buffer-compare-instance-method.js

```diff
@@ -2,8 +2,8 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  size: [16, 512, 1024, 4096, 16386],
-  args: [1, 2, 3, 4, 5],
+  size: [16, 512, 4096, 16386],
+  args: [1, 2, 5],
```

Comment:
The size argument should behave mostly identical and it seems the values where chosen somewhat random. It was mainly to verify that small and big values work properly. The middle part is normally not required since the algorithm should show a curve depending on the size.

The same as here also applies to the next few benchmarks when it comes to the size.

I also removed two `args` cases: these just test some javascript code paths that are mostly negligible. The main performance cost is in the C++ layer and as long as no args and all args do not show significant different numbers (which is not visible by the summary but only by the individual benchmark run outcome!) everything works as expected (that does seem to be the case).